azure_arc_servers_jumpstart/proxy/artifacts/proxy_setup.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Update
+
+sudo apt-get clean
+sudo apt-get update
+
+wget -qO - https://packages.diladele.com/diladele_pub.asc | sudo apt-key add -
+
+# add new repo
+echo "deb https://squid55.diladele.com/ubuntu/ focal main" | sudo tee -a /etc/apt/sources.list.d/squid55.diladele.com.list
+
+# and install
+sudo apt-get update && sudo apt-get install -y \
+    squid-common \
+    squid-openssl \
+    squidclient \
+    libecap3 libecap3-dev
+
+sudo systemctl stop apparmor
+sudo systemctl disable apparmor
+sudo systemctl enable squid
+
+
+# Generating the certificate and key
+sudo openssl req -new -newkey rsa:2048 -sha256 -days 365 -nodes -x509 -extensions v3_ca -keyout /tmp/squid-ca-key.pem -out /tmp/squid-ca-cert.pem -subj "/C=US/ST=Washington/L=Redmond/O=SecurArcity/OU=Hybrid/CN=www.jumpstart.com"
+sudo cat  /tmp/squid-ca-cert.pem /tmp/squid-ca-key.pem >> /tmp/squid-ca-cert-key.pem
+sudo openssl x509 -in /tmp/squid-ca-cert-key.pem -outform DER -out /tmp/squidTrusted.der
+
+# create a directory
+sudo mkdir -p /etc/squid/certs
+# Move certificate to the directory
+sudo cp /tmp/squid-ca-cert-key.pem /etc/squid/certs/
+# change ownership so that squid can access the certificate
+sudo chown proxy:proxy -R /etc/squid/certs
+# modify certificate permissions (for self-signed certificate)
+sudo chmod 700 /etc/squid/certs/squid-ca-cert-key.pem
```

Comment:
```suggestion
# Create a directory
sudo mkdir -p /etc/squid/certs
# Move certificate to the directory
sudo cp /tmp/squid-ca-cert-key.pem /etc/squid/certs/
# Change ownership so that squid can access the certificate
sudo chown proxy:proxy -R /etc/squid/certs
# Modify certificate permissions (for self-signed certificate)
sudo chmod 700 /etc/squid/certs/squid-ca-cert-key.pem
```